l10n_ch_fds_postfinance/tests/test_fds_keys.py

```diff
@@ -0,0 +1,103 @@
+# © 2017 Compassion CH <http://www.compassion.ch>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import mock
+
+from odoo.exceptions import Warning as UserError
+from odoo.tests.common import TransactionCase
+
+# auth_keys = 'openerp.addons.l10n_ch_fds_postfinance.models' \
+#     '.fds_authentication_keys.FdsAuthenticationKeys'
+
+
+# class TestFds(TransactionCase):
```

Comment:
I did not work a lot on V11 migrations, but it seems like a bad idea to just delete tests.
Is there any reason to do so?

Reply:
I can't accept this PR with the tests commented, please reactivate them or create new ones.